domain/model/service/service.go

```diff
@@ -84,6 +85,11 @@ type State interface {
 	// returned.
 	ListModelsForUser(context.Context, coreuser.UUID) ([]coremodel.Model, error)
 
+	// GetModelUsers will retrieve basic information about all users with
+	// permissions on the given model UUID.
+	// If the model cannot be found it will return modelerrors.NotFound.
+	GetModelUsers(ctx context.Context, modelUUID coremodel.UUID) ([]coremodel.ModelUserInfo, error)
```

Comment:
**thought:** Oh. The state doesn't expose the filter... So it's a service layer stuff. So you could definitely pass a well defined filter to Service layer.